t/full-stack.t

```diff
@@ -33,6 +33,7 @@ BEGIN {
 
 use FindBin;
 use lib "$FindBin::Bin/lib", "$FindBin::Bin/../external/os-autoinst-common/lib";
+use Mojo::Base -signatures;
```

Comment:
Since this activates a lexical feature for the whole namespace (similar to `strict`/`warnings`), should this go to the top, near `use Test::Most`?

Reply:
Oh wait, there's also `autodie` already below, that's pretty much the same case.

Reply:
The pattern is/was to use `Test::Most` first, then comes `FindBind` and only then other modules (because they might depend on the setup done via `FindBind`). So considering these rules I've included it as soon as possible.

Reply:
Not really: https://github.com/os-autoinst/openQA/blob/master/docs/Contributing.asciidoc#code-style-suggestions says `"strict", "warnings" or other modules that provide static checks` and also put Mojo::Base *before* `use lib`

Reply:
`-signatures` is neither "strict", "warnings" and also doesn't provide any static checks. (It enables a lexer feature which most code does not need.)

Reply:
Actually it does provide static checks, because signatures require the code to be parsed differently. Different things become syntax errors.

Reply:
We should probably just consider `use feature` (and all derivatives) one level below `use warnings`.